### PR TITLE
Add application bake settings

### PR DESCRIPTION
**Pull request by slimbuck**
Add support for bake parameters to the application render settings.

### CODE REVIEW

File: src/scene/scene.js [hunk 1]
```diff
@@ -596,27 +596,43 @@ class Scene extends EventHandler {
...
+        this.lightmapMode = render.lightmapMode;
+        this.exposure = render.exposure;
+        this._skyboxIntensity = render.skyboxIntensity === undefined ? 1 : render.skyboxIntensity;
+        this._skyboxMip = render.skyboxMip === undefined ? 0 : render.skyboxMip;
+
+        if (render.skyboxRotation) {
+            this._skyboxRotation.setFromEulerAngles(render.skyboxRotation[0], render.skyboxRotation[1], render.skyboxRotation[2]);
         }
 
+        const apply = (name) => {
```
Comment (mvaligursky) [inline src/scene/scene.js:623]: maybe call this `applyRender`? And apply to the rest of this function.

File: src/scene/scene.js [hunk 2]
```diff
@@ -596,27 +596,43 @@ class Scene extends EventHandler {
...
+            }
+        };
+
+        // bake
+        apply('ambientBake');
+        apply('ambientBakeNumSamples');
+        apply('ambientBakeSpherePart');
+        apply('ambientBakeOcclusionBrightness');
+        apply('ambientBakeOcclusionContrast');
+
```
Comment (willeastcott) [inline src/scene/scene.js:635]: Not that you have to address this now, but I just want to point out that I'm not keen on `applySettings` being on the `Scene` class. This is really just a utility function for when serialized scene settings are loaded in. So I'd put this wherever that happens. What do other people think?

Comment (slimbuck) [reply src/scene/scene.js:635]: I agree. This is actually a detail of the application (whence this function is called), not the scene.

File: src/scene/scene.js (hunk 1 above)
Comment (slimbuck) [reply src/scene/scene.js:623]: I thought about that, but actually only `skyboxIntensity` and `skyboxMip` work the same way, so figured I'd leave it.

**Pull request merged**